test/integration/consul-container/libs/cluster/cluster.go

```diff
@@ -341,6 +341,27 @@ func (c *Cluster) PeerWithCluster(acceptingClient *api.Client, acceptingPeerName
 	return nil
 }
 
+func (c *Cluster) ConfigEntryWrite(entryStr string) error {
+	client, _ := c.GetClient(nil, true)
+
+	entry, err := helpers.ParseConfigEntry(entryStr)
+	fmt.Println("Config entry kind", entry.GetKind())
+	if err != nil {
+		return fmt.Errorf("error parse config entry string")
+	}
+
+	entries := client.ConfigEntries()
+	written := false
+	written, _, err = entries.Set(entry, nil)
+	if err != nil {
+		return fmt.Errorf("error set config entry: %v", err)
+	}
+	if !written {
+		return fmt.Errorf("config entry not updated: %s/%s", entry.GetKind(), entry.GetName())
+	}
+	return err
```

Comment:
I think you should return `nil` here instead of err. Potential errors has been handled above in L350, 357 and 360